### ISSUE TITLE
Is integram.org having issues?

### ISSUE THREAD

**Issue by brunofm** [requilence/integram#130]
Right now I can't consistently open https://integram.org/ (sometimes it loads, sometimes it times out) and my GitLab webhooks only give a 502 error:

<img width="382" alt="image" src="https://user-images.githubusercontent.com/7012854/233970108-f5c1e55f-e1b4-4daf-9d63-fc8f4ece2d01.png">

Is anyone else having similar issues?

**Comment by brunofm** [comment]
After a few times retrying, now I got a new error in GitLab:

<img width="311" alt="image" src="https://user-images.githubusercontent.com/7012854/233970543-d7c91ea4-46f7-4e45-8a7c-73f591d19108.png">

**Comment by mhkafadar** [comment]
@brunofm I am also a user of Integram, but it has been down for a while. In the meantime, I've developed an open-source alternative that you can use for GitHub and GitLab - Trello is not supported for now.

You can use this alternative until Integram is back up and running.

https://github.com/mhkafadar/notifine

**Comment by nicexe2e4** [comment]
@mhkafadar thanks for your work, but without MR and Pipe notifications it's useless(

**Comment by gituser** [comment]
@nicexe2e4 use self-hosted integram version and there will be no issues

**Comment by nicexe2e4** [comment]
@gituser yeah, I want to try to deploy this in my k8s, just today I ran into a notification task, ty.

**Comment by gituser** [comment]
@nicexe2e4 there is no need for k8s, you're overcomplicating things.
Just use simple container lxc or docker or kvm machine.
Basically you need to run at least 2 or more binaries, one for integram (core service), one for gitlab (https://github.com/integram-org/gitlab) or trello (https://github.com/integram-org/trello) or any other service to get notifications.

Configuration is controlled by ENV variables.

**Comment by mhkafadar** [comment]
> @mhkafadar thanks for your work, but without MR and Pipe notifications it's useless( 

@nicexe2e4 

Thank you for your feedback. I'm happy to let you know that I've implemented MR and Pipeline notifications for Gitlab. Please give it a try, and for any future suggestions or feedback, feel free to open an issue on the repository. I'll make sure to address and implement any required features within 3 days after the issue is opened.

**Comment by adibaulia** [comment]
> @nicexe2e4 there is no need for k8s, you're overcomplicating things. Just use simple container lxc or docker or kvm machine. Basically you need to run at least 2 or more binaries, one for integram (core service), one for gitlab (https://github.com/integram-org/gitlab) or trello (https://github.com/integram-org/trello) or any other service to get notifications.
> 
> Configuration is controlled by ENV variables.

how to install it? i need to use the bitbucket for telegram bot. the documentation for installing quite hard to read

**Comment by gituser** [comment]
@adibaulia 

You need to build 2 binaries:
* one from this repository - https://github.com/requilence/integram
* another one from this - https://github.com/integram-org/bitbucket

Follow installation instructions in each of the repositories, you'll figure it out eventually :)

**Comment by brunofm** [comment]
This week I randomly started getting notifications in our Telegram group for things that happened back when I opened this issue. Got probably dozens or even hundreds of notifications throughout the week.

Has this happened to anyone else?
